pkg/process/checks/pod.go

```diff
@@ -209,5 +213,108 @@ func extractPodMessage(p *v1.Pod) *model.Pod {
 		podModel.ContainerStatuses = append(podModel.ContainerStatuses, &cStatus)
 	}
 
+	podModel.KubectlStatus = computeKubectlStatus(p)
+	podModel.ConditionMessage = getConditionMessage(p)
+
 	return &podModel
 }
+
+// mostly copied from kubernetes to match what users see in kubectl
+// in case of issues, check for changes upstream: https://github.com/kubernetes/kubernetes/blob/1e12d92a5179dbfeb455c79dbf9120c8536e5f9c/pkg/printers/internalversion/printers.go#L685
+func computeKubectlStatus(p *v1.Pod) string {
```

Comment:
Thoughts on extracting this logic into `pkg/` somewhere? Seems like it's pretty agnostic to this specific check code anyways.

Reply:
Aside: Some unit tests would be good too :)